From 3a474a948b1c6ac3dc09b991e41ef2e6b65658c6 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Tue, 2 Jul 2024 13:53:31 +0200
Subject: [PATCH 0996/1695] clk: stm32mp21: add clock calibration for HSI / MSI
 clocks

HSI monitoring control, with LSE, is based on the RCC HSI Frequency
Monitoring Control Register (RCC_HSIFMONCR) and RCC HSI Frequency
Value Register (RCC_HSIFVALR).
MSI monitor is based on the clock frequency calculator.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 50c4893c2e90b8f081a24b101f643c5fc2f1d66a clk: stm32mp21: add clock calibration for HSI / MSI clocks]
Change-Id: Ie0dbc98c4430e8894dfba8f7bc2825fdb81f1cbd
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/390830
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/drivers/clk/sub.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/drivers/clk/sub.mk b/core/drivers/clk/sub.mk
index 4ffd1b0b0..d2e988e5f 100644
--- a/core/drivers/clk/sub.mk
+++ b/core/drivers/clk/sub.mk
@@ -4,7 +4,7 @@ srcs-$(CFG_DRIVERS_CLK_FIXED) += fixed_clk.c
 srcs-$(CFG_STM32MP_CLK_CORE) += clk-stm32-core.c
 srcs-$(CFG_STM32MP13_CLK) += clk-stm32mp13.c clk-stm32-rcc.c
 srcs-$(CFG_STM32MP15_CLK) += clk-stm32mp15.c clk-stm32-rcc.c
-srcs-$(CFG_STM32MP21_CLK) += clk-stm32mp21.c clk-stm32-rcc.c
+srcs-$(CFG_STM32MP21_CLK) += clk-stm32mp21.c clk-stm32-rcc.c clk-stm32mp2-cal.c
 srcs-$(CFG_STM32MP25_CLK) += clk-stm32mp25.c clk-stm32-rcc.c clk-stm32mp2-cal.c
 
 subdirs-$(CFG_DRIVERS_SAM_CLK) += sam
-- 
2.47.2

